canonical-websites/www.ubuntu.com
# Add new page - /legal/data-privacy/snapcraft-nps

## Done

- created the page
- added it to the nav
- updated and moved the [copy doc](https://docs.google.com/document/d/1JODNGJQ0G-aSrycKTv4XOGmJVWw98Fqk2gLfHmQqgB4/edit)
- made the base layout generic and move the UASD one to that folder

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/legal/data-privacy/snapcraft-nps](http://0.0.0.0:8001/legal/data-privacy/snapcraft-nps)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that it is in the navigation
- See that the copy is correct per the copy doc

## Issue / Card

Fixes #4526
